.github/workflows/docker-arm64.yaml

```
@@ -19,22 +19,25 @@ jobs:
     steps:
       - name: Checkout code
         uses: actions/checkout@v2
```

actionlint (1.7.4)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)